Sources/SnowManLogic/ViewLogic/UI.swift

```diff
@@ -0,0 +1,12 @@
+public class UI: Receivable {
+	public init() {
+	}
+
+	public func getUserInput() -> String {
+		return String(readLine(stripNewline: true) ?? "")
```

Comment:
this is very hard to test. The `UI` implementation should just wrap the interface of the IO functions, so `getUserInput` to `readLine` relationship should look like the `display` to `print` relationship does.